boards/arm/qemu_mps2/qemu_mps2.dts

```diff
@@ -0,0 +1,107 @@
+/dts-v1/;
+
+#include <arm/armv7-m.dtsi>
+
+/ {
```

Comment:
So, why we duplicate all this, instead of #inlcude'ing it?

Reply:
including it from where?

Reply:
From real mps_an385 DTS.

Reply:
it is not an exact copy, we have removed anything that does not apply to running in qemu.

Reply:
Well, seems like we're just eager to complicate our own life here. If the only (?) problem with mps2_an385 vs qemu variant of it is the need to disable tickless mode, why do more work than that?